client/src/services/textFormat.js

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+
+// SPDX-License-Identifier: MIT
+
+const formatDate = (date) => {
+  const string = date.toLocaleString('en-GB').replaceAll('/', '.');
```

Comment:
I would guess that this is using the locale `en-GB` and replacing `/` with `.` rather than just using `fi` because `fi` seems to format the time as `9.30` and we don't like that. May be a good idea to add a comment explaining this.